beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -156,6 +156,31 @@
     - timba
     - twoubadou
     - zouk
+- classical:
+    - ballet
+    - baroque:
+        - baroque music
+    - cantata
+    - chamber music:
+        - string quartet
+    - classic
```

Comment:
Oh, I see what you mean. I don't think that's the right place for `classic`, wouldn't you end up having things like `classic rock` being tagged as `classical` by mistake then?